app/routes/invitations.rb

```diff
@@ -3,11 +3,13 @@ module Routes
     class Invitations < Core
       post '/teams/:slug/invitations' do |slug|
         please_login
+        team = Team.find_by(slug: params[:slug])
+        valid_invitee = (params[:usernames].split(",") - team.members.map(&:username)).join(",")
         only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
-          team.invite_with_usernames(params[:usernames], current_user)
+          team.invite_with_usernames(valid_invitee, current_user)
           team.save
 
-          flash[:success] = "Invitation sent to team #{slug}"
+          flash[:success] = "Invitation sent to team #{slug}" if valid_invitee.size > 0
```

Comment:
I will add notifications for that part, saying "user xxx is already in the team". Thanks for pointing it out.